app/store/middlewares/helpers.js

```diff
@@ -264,38 +136,15 @@ export function cleanUpState(payload, keepCurrent = false) {
         nextEntities.posts.pendingPostIds = nextPendingPostIds;
     }
 
-    const nextState = {
-        app: resetPayload.app,
-        entities: {
-            ...nextEntities,
-            channels: payload.entities.channels,
-            emojis: payload.entities.emojis,
-            general: resetPayload.entities.general,
-            preferences: resetPayload.entities.preferences,
-            search: {
-                ...resetPayload.entities.search,
-                results: searchResults,
-                flagged: flaggedPosts,
-            },
-            teams: resetPayload.entities.teams,
-            users: payload.entities.users,
-            roles: resetPayload.entities.roles,
-        },
-        views: {
-            announcement: payload.views.announcement,
-            ...resetPayload.views,
-            channel: {
-                ...resetPayload.views.channel,
-                ...payload.views.channel,
-            },
-        },
-        websocket: {
-            lastConnectAt: payload.websocket?.lastConnectAt,
-            lastDisconnectAt: payload.websocket?.lastDisconnectAt,
-        },
+    nextState.views.root = {
+        hydrationComplete: true,
```

Comment:
this I don't like cause it can make the app believe hydration is complete when in fact is not, will definitely impact the upgrade from 1.30 to 1.31